hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsTableLatencies.java

```diff
@@ -16,13 +16,14 @@
  */
 package org.apache.hadoop.hbase.regionserver;
 
+import org.apache.hadoop.hbase.metrics.BaseSource;
 import org.apache.yetus.audience.InterfaceAudience;
 
 /**
  * Latency metrics for a specific table in a RegionServer.
  */
 @InterfaceAudience.Private
-public interface MetricsTableLatencies {
+public interface MetricsTableLatencies extends BaseSource {
```

Comment:
MetricsTableLatencies extends BaseSourceImpl which implements BaseSource.
Can't see the reason why MetricsTableLatencies extends this interface, again.